crates/common/Cargo.toml

```diff
@@ -14,5 +14,5 @@ doctest = false
 slight-events-api = { path = "../events-api" }
 slight-http-api = { path = "../http-api/" }
 as-any = "0.3"
-wasmtime = "0.35"
+wasmtime = "0.39.1"
```

Comment:
Why use `0.39.1` instead of `0.39`? Across all other deps in the project, we've been sticking to only `major.minor` semver

Reply:
good point!